tests/e2e/velero_deployment_suite_test.go

```diff
@@ -0,0 +1,345 @@
+package e2e
+
+import (
+	"log"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
+	velero "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/pointer"
+)
+
+var _ = Describe("Configuration testing for Velero Custom Resource", func() {
+
+	var _ = BeforeEach(func() {
```

Comment:
I'm wondering if this would be better in `BeforeSuite`. This way we create the secret before the test suite runs, and then we don't need to keep creating it before each test. What do you think?

Reply:
Makes sense. It should be a part of initial setup.